Implement container start lifecycle operation

KKoukiou opened the pull request.

KKoukiou commented:
If I delete the container from cli, since we don't catch events state is not updated in cockpit-podman. So trying to start the deleted container will result in the follwing error.
![container-start-error](https://user-images.githubusercontent.com/14921356/52221947-a041f180-28a2-11e9-837f-6072ed586242.png)

larskarlitski commented:
Yep, this needs a rebase and this patch to make it work with the new API:
```diff
diff --git a/src/Containers.jsx b/src/Containers.jsx
index 5e0b894..7a14d30 100644
--- a/src/Containers.jsx
+++ b/src/Containers.jsx
@@ -66,10 +66,10 @@ class Containers extends React.Component {
     }
 
     startContainer(container) {
-        utils.varlinkCall(utils.PODMAN, "io.podman.StartContainer", { name: container.Name })
+        varlink.call(utils.PODMAN_ADDRESS, "io.podman.StartContainer", { name: container.name })
                 .then(() => this.props.updateContainersAfterEvent())
                 .catch(ex => this.setState({
-                    actionError: cockpit.format(_("Failed to start container $0"), container.Name),
+                    actionError: cockpit.format(_("Failed to start container $0"), container.name),
                     actionErrorDetail: ex
                 }));
     }
```

Thanks!

KKoukiou commented:
I implemented extra to start operation, the restart, force restart, stop, force stop. I fixed the unclickable dropdown and add the error notification with posibility to dismiss it.
Operations are covered by a new test function.
Screenshots follow.
![podman-restart-error](https://user-images.githubusercontent.com/14921356/52271251-6c6ad880-2943-11e9-8bb4-b7f87558640d.png)
![podman-stop-restart](https://user-images.githubusercontent.com/14921356/52271252-6d036f00-2943-11e9-9a05-aa276e05afa9.png)
![podman-start](https://user-images.githubusercontent.com/14921356/52271253-6d036f00-2943-11e9-9c2f-d37331470350.png)
